Dockerfile.base

```diff
@@ -78,7 +78,8 @@ RUN apt-get -y update && \
 		valgrind \
 		wget \
 		ovmf \
-		xz-utils
+		xz-utils \
+		thrift-compiler
```

Comment:
Is this all that is required to build Thrift tests and samples in the CI?

Reply:
Yes. If the user wants to run the host-side demos, then there are additional things to install, but they are not required for simply building the Zephyr tests.